webview/src/shared/components/sectionContainer/InfoTooltip.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import cx from 'classnames'
+import { PlotsSection } from 'dvc/src/plots/webview/contract'
+import { SetupSection } from 'dvc/src/setup/webview/contract'
+import styles from './styles.module.scss'
+import { SectionDescription } from './SectionContainer'
+import Tooltip from '../tooltip/Tooltip'
+import { Icon } from '../Icon'
+import { Info, PassFilled, Error } from '../icons'
+
+export enum TooltipIconType {
+  PASSED = 'icon-pass-filled',
+  INFO = 'icon-info',
+  ERROR = 'icon-error'
+}
+
+const getIcon = (icon?: TooltipIconType) => {
+  const props = {
+    className: styles.infoIcon,
```

Comment:
By using having `TooltipIconType` match the icon directly, I think you could move all props directly on the component and use `cx()` with conditionals.